### PR TITLE
Add share helpers

### PR DESCRIPTION
**Pull request by convolu**

### CODE REVIEW

**Comment by convolu** [comment]
This is something that we seem to replicate in our apps every time we need to share. Provide a better interface that doesn't require knowledge of the implementation. This could be the start of a better api for the kano_world fns.

cc @tombettany @radujipa @alex5imon @pazdera @Ealdwulf @skarbat  Thoughts?

**Comment by tombettany** [comment]
Long overdue. :+1:

File: kano_world/share_helpers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from kano.utils.shell import run_cmd
+from .functions import login_using_token
+from .share import upload_share
+
+
+def login_and_share(file_path, title, app_name):
+    """ Make a share to Kano World. If the user is not logged in, present the
+    register/login window to the user.
+    :param file_path: Code file to be uploaded to Kano World
+    :type str:
```
Comment (pazdera) [inline kano_world/share_helpers.py]: The variable name is missing here.

I'd also format the comment a little to make it a bit more readable (but that's just my personal preference):

https://github.com/KanoComputing/kano-init/blob/master/kano_init/user.py#L34

**Comment by pazdera** [comment]
Looks good, just a small comment to the docstring.

:shipit: otherwise

**Comment by convolu** [comment]
Fixed the docstring as per your comment, @pazdera.

Merging for testing, thanks

**Pull request merged**